src/css/components/_captions-settings.scss

```diff
@@ -21,6 +21,40 @@
   vertical-align: bottom;
 }
 
+// code that will only run if CSS Grid is supported by the browser
+@supports (display: grid) {
+  .vjs-text-track-settings .vjs-modal-dialog-content {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr auto;
+  }
+
+  .vjs-text-track-settings .vjs-track-settings-colors {
+    display: block;
+    grid-column: 1;
+    grid-row: 1;
+  }
+
+  .vjs-text-track-settings .vjs-track-settings-font {
+    grid-column: 2;
+    grid-row: 1;
+  }
+
+  .vjs-text-track-settings .vjs-track-settings-controls {
+    grid-column: 2;
+    grid-row: 2;
+
+  }
+
+  .vjs-text-track-settings fieldset span.vjs-opacity {
+    margin-left: unset;
```

Comment:
instead of having this and the margin-right, should we just update the default styling below to have a margin-right?

Reply:
Just tried it out and it works good in both the grid and table moving this to the default style.